packages/manager/modules/iam/src/components/deleteEntity/translations/Messages_fr_FR.json

```diff
@@ -12,6 +12,12 @@
   "iam_delete_entity_resourceGroup_name": "Suppression du groupe de ressources : ",
   "iam_delete_entity_resourceGroup_success": "Le groupe de ressources a bien été supprimé",
   "iam_delete_entity_resourceGroup_warn": "Êtes-vous sûr.e de confirmer la suppression ?",
+  "iam_delete_entity_application_error": "An error occurred while deleting the application: {{ message }}",
+  "iam_delete_entity_application_field": "Please enter the word \"{{ statement }}\" to delete your application",
+  "iam_delete_entity_application_heading": "Delete an application",
+  "iam_delete_entity_application_name": "Deleting application: ",
+  "iam_delete_entity_application_success": "Application deleted",
+  "iam_delete_entity_application_warn": "Warning: this action will delete the application and it will revoke all credential belonging to this application.",
```

Comment:
It's normal the text is in english ?

Reply:
yes, i'll fix the translations content on a regular PR which will be based on this PR target branch

Reply:
Will be fixed in a next PR when it's merged